datanode/src/storage_api.py

```diff
@@ -292,17 +297,86 @@ def GridCellsOperationDataHandler(zoom, gufi):
   return _FormatResult(result)
 
 
+@webapp.route(
+  '/UVR/<zoom>/<message_id>',
+  methods=['PUT', 'POST', 'DELETE'])
+def UvrDataHandler(zoom, message_id):
+  """Handles the web service request for emplacing or removing a UVR.
+
+  Args:
+    zoom: zoom level in slippy tile format
+    message_id: unique message ID of constraint
+    OAuth access_token as part of the header
+  Returns:
+    200 with token and metadata in JSON format,
+    or the nominal 4xx error codes as necessary.
+  """
+  uss_id = _ValidateAccessToken(UVR_WRITE_SCOPE)
+
+  if not uss_id:
+    return {
+      'status': 'fail',
+      'code': status.HTTP_400_BAD_REQUEST,
+      'message': 'uss_id must be provided in the request to modify a UVR.'}
+
+  uvr_json = _GetRequestParameter('uvr', None)
+  if not uvr_json:
+    return {
+      'status': 'error',
```

Comment:
Status should be "fail" if the problem is with the data submitted. https://labs.omniti.com/labs/jsend

Reply:
This was modeled off of lines 516, 610, 728, and 819 (PUT Operator(s) or Operation(s) fails with 'error' if missing fields).  Should I change those as well for consistency?

Reply:
Ah, I see. I think we'll make everything more consistent with #19 and #20.